riot.im/release/config.json

```diff
@@ -24,8 +24,7 @@
         "policyUrl": "https://matrix.org/legal/riot-im-cookie-policy"
     },
     "features": {
-        "feature_cross_signing": "enable",
-        "feature_event_indexing": "enable"
+        "feature_cross_signing": "enable"
```

Comment:
isn't it no longer a feature?

Reply:
Correct, though no one ever updated riot-desktop :(